src/Entity/User.php

```diff
@@ -265,6 +266,11 @@ public function getUsername(): string
         return (string) $this->email;
```

Comment:
Maybe it could be removed?

Reply:
I tried but the UseInterface still requires this method, same with `getSalt()`.
We ill be able to remove it in 5.4 if I remember correctly, or 6.0 maybe